layouts/pages/index.ejs

```diff
@@ -12,9 +12,6 @@
     <div id="main-search-page">
       <div class="main-item">
         <%- include('partials/branding', {context: 'home'}) %>
-        <div class="tagline">
-          <p><%- template('landing.tagline') %> <%- template('landing.quickLink') %></p>
-        </div>
```

Comment:
I'm down to leave this code in and just live with extra vertical space, to reduce how much we diverge from upstream. (Caveat: I don't remember how much vertical space this was, and maybe I will hate it so much after seeing it that I want to remove it after all. 😛)

Reply:
Well, looking at those two changes together, I see that it's not exactly a layout change since the stylesheet was preventing this element from being displayed at all. But I still feel like it doesn't belong in this PR.

Reply:
It's needed for this PR, because without it if you fail login there's no obvious way to log out unless you know the magic logout link. It just takes you to a blank page.

I can separate this into a prerequisite PR if you'd like, it's just not an optional change.

Reply:
@redfern314 Sorry, I'm still confused. It looks to me like the magic logout link is added in `userTools.ejs`, unrelated to the change here.

Reply:
The userTools.ejs change actually doesn't show up on the error screen because the user isn't technically logged in. This change makes the 403 message inviting the user to re-log-in in my second screenshot show up, without this change it's just a blank page body.

Reply:
Moved this change to #5.